rafay/resource_addon.go

```diff
@@ -37,6 +43,38 @@ func resourceAddon() *schema.Resource {
 	}
 }
 
+func resourceAddonImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	idParts := strings.SplitN(d.Id(), "/", 3)
+	log.Println("resourceAddonImport idParts:", idParts)
+	d_debug := spew.Sprintf("%+v", d)
+	log.Println("resourceAddonImport d.Id:", d.Id())
+	log.Println("resourceAddonImport d_debug", d_debug)
+
+	if len(idParts) > 2 && idParts[2] != "" && idParts[2] == "show_artifact" {
```

Comment:
do we need `idParts[2] != ""` check also? `== "show_artifact"` is sufficient right?

Reply:
There is possible, we may get panic error if they not added artifact after last `/` .. like below `resourceName/projectName/`